README.md

```diff
@@ -59,135 +60,66 @@ with sycl_context("gpu"):
-```
-
-# Building daal4py without conda-build
-Without conda-build you have to manually setup your environment before building daal4py.
-
-## Prerequisites
-* Python version >= 3.6
-* Jinja2
-* Cython
-* Numpy
-* A C++ compiler with C++11 support
-* Intel® Data Analytics Acceleration Library (Intel® DAAL) version 2019 or later (https://github.com/01org/daal)
-  * You can use the pre-built conda package from Intel channel on anaconda.org
-* MPI
-  * You can use the pre-built conda package intel or conda-forge channel on anaconda.org
-  * Needed for distributed mode. You can disable support for distributed mode by setting NO_DIST to '1' or 'yes'
-For oneAPI support
-* A DPC++ compiler 
-* Intel(R) oneAPI Data Analytics Library version 2021.1 or later (https://oneapi-src.github.io/oneDAL/)
-  * You can use the pre-built conda package from Intel channel on anaconda.org
-
-## Setting up a build environment
-The easiest path for getting cython, oneDAL, MPI etc. is by creating a conda environment and setting environment variables:
-```
-conda create -n DAAL4PY python=3.7 impi-devel daal daal-include cython jinja2 numpy clang-tools -c intel -c conda-forge
-conda activate DAAL4PY
-export DALROOT=$CONDA_PREFIX
-export MPIROOT=$CONDA_PREFIX
-```
-
-## Configuring the build with environment variables
-* DAAL4PY_VERSION: sets package version
-* NO_DIST: set to '1', 'yes' or alike to build without support for distributed mode
-* NO_STREAM: set to '1', 'yes' or alike to build without support for streaming mode
-
-### Notes on building for macOS
-If building in High Sierra or higher, one may have to run into C++ build errors related to platform targets. Utilize ```export MACOSX_DEPLOYMENT_TARGET="10.9"``` if running into platform target issues.
-
-## Building daal4py
-Requires Intel(R) oneAPI Data Analytics Library and Intel(R) MPI Library being properly set up, meaning you have to set DALROOT and MPIROOT variables.
-```bash
-cd <checkout-dir>
-python setup.py build_ext
-```
-
-## Installing daal4py
-Requires Intel(R) oneAPI Data Analytics Library and Intel(R) MPI Library being properly set up, meaning you have to set DALROOT and MPIROOT variables.
-```bash
-cd <checkout-dir>
-python setup.py install
-```
-
-# Building documentation for daal4py
-## Prerequisites for creating documentation
-* sphinx
-* sphinx_rtd_theme
-
-## Building documentation
-1. Install daal4py into your python environment
-2. ```cd doc && make html```
-3. The documentation will be in ```doc/_build/html```
+result = init.compute(X, daal4py.my_procid())
+d4p.daalfini()
+```
+
+# Scikit-learn patching
+
+| *Speedups of daal4py-powered Scikit-learn over the original Scikit-learn* |
+|:--:|
+| ![](doc/IDP%20scikit-learn%20accelearation%20compared%20with%20stock%20scikit-learn.png) |
+| *Technical details: FPType: float32; HW: Intel(R) Xeon(R) Platinum 8280 CPU @ 2.70GHz, 2 sockets, 28 cores per socket; SW: scikit-learn 0.23.1, Intel® oneDAl (2021.1 Beta 10)* |
```

Comment:
Are you sure that float32 and not float64?

Reply:
fixed